### PR TITLE
Correct Model Docstrings and Make Models PEP 8 Compliant

### PR DESCRIPTION
**Pull request by PaulHuwe**
Changes:
- Made datamodels PEP 8 compliant. Removed remaining zeroframe and pathloss from models. 
- Added variance schema for image and rampoutput files. 
- Removed a seemingly incorrectly named file. @dmggh can you confirm that keyword_gainfact.schema is extraneous (given that we use keyword_gainfact.schema.yaml)?

Tests:
- Datamodels tests did not require updating. These changes passed those tests.

Documentation:
- This ticket is primarily an update to (as yet not fully supported) documentation.

Ticket:
- RCAL-99

### CODE REVIEW

**Comment by dmggh** [comment]
Yes, keyword_gainfact.schema is extraneous and can be removed.

**Comment by ddavis-stsci** [comment]
Why can we remove gain factor? in https://innerspace.stsci.edu/display/ROMAN/Use+Cases+for+Level+2+Science+Data+Processing
it is listed in the steps, 2k on the "Standard Path".

File: romancal/datamodels/dynamicdq.py [hunk 1]
```diff
@@ -25,7 +27,8 @@ def dynamic_mask(input_model):
             try:
                 standard_bitvalue = dqflags.pixel[dqname]
             except KeyError:
-                log.warning('Keyword %s does not correspond to an existing DQ mnemonic, so will be ignored' % (dqname))
+                log.warning('Keyword %s does not correspond to an existing DQ mnemonic, '
+                            'so will be ignored' % dqname)
```
Comment (ddavis-stsci) [inline romancal/datamodels/dynamicdq.py:31]: Should we change keyword to something else? DQ flag?

**Review by ddavis-stsci — CHANGES_REQUESTED** [review]
We need to resolve the gain factor.

**Review by ddavis-stsci — APPROVED** [review]
TGTM

File: romancal/datamodels/dynamicdq.py (hunk 1 above)
Comment (PaulHuwe) [reply romancal/datamodels/dynamicdq.py:31]: dqflags is already used, so I think qdname should stay as is.

**Pull request merged**